onward/app/views/fragments/rightMostPopular.scala.html

```diff
@@ -18,7 +19,14 @@ <h3 class="content__meta-heading">Most popular</h3>
                                 <img class="responsiveimg" src="@ImgSrc.getFallbackUrl(image.images)" alt="" />
                             </div>
                         }
-                        <h4 class="right-most-popular-item__headline media__body">@trail.properties.linkText</h4>
+                        <div class="media__body">
+                            <h4 class="right-most-popular-item__headline">@trail.properties.linkText</h4>
+                            @trail.properties.maybeContent.map { content =>
+                                @if(content.tags.tags.exists(_.id == "tone/news")) {
+                                    @fragments.contentAgeNotice(ContentOldAgeDescriber(content))
+                                }
+                            }
+                        </div>
```

Comment:
`rightMostPopular.scala.html` template doesn't seem to be in use anymore. However I still included this change in it just in case.